webpack/set-webpack-public-path-plugin/src/SetPublicPathPlugin.ts

```diff
@@ -126,7 +126,7 @@ const ASSET_NAME_TOKEN: string = '-ASSET-NAME-c0ef4f86-b570-44d3-b210-4428c5b782
 const ASSET_NAME_TOKEN_REGEX: RegExp = new RegExp(ASSET_NAME_TOKEN);
 
 function escapeRegExp(str: string): string {
-  return str.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+  return str.replace(/[.*+?^${}()|[\]\\\/]/g, '\\$&');
```

Comment:
Performance Nit: Prefer hoisting Regular Expressions to top of file vs using them as Literals. RexExp literals cause additional GC